### PR TITLE
Removed usage of --no-sound-null-safety flag.

### PR DESCRIPTION
**Pull request by mannprerak2**
The `--no-sound-null-safety` flag was causing an issue when package was being run, since it is first precompiled. See comments on #128.
This PR fixes that, it follows the approach of the package:test => https://github.com/dart-lang/test/blob/master/pkgs/test/bin/test.dart.

### CODE REVIEW

File: lib/src/executables/setup.dart [hunk 1]
```diff
@@ -0,0 +1,275 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
```
Comment (dcharkes) [inline lib/src/executables/setup.dart:1]: 2021

File: lib/src/executables/ffigen.dart [hunk 2]
```diff
@@ -0,0 +1,211 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
```
Comment (dcharkes) [inline lib/src/executables/ffigen.dart:1]: 2021. In the Dart project we usually use the year that we add the file (not the year we started the project).

**Review by dcharkes — APPROVED** [review]

File: tool/coverage.sh [hunk 3]
```diff
@@ -11,7 +11,7 @@ set -e
 pub global activate remove_from_coverage
 pub global activate coverage
 # Generate coverage report.
-dart --no-sound-null-safety --pause-isolates-on-exit --disable-service-auth-codes --enable-vm-service=3000 test/test_coverage.dart &
+dart --no-sound-null-safety  --pause-isolates-on-exit --disable-service-auth-codes --enable-vm-service=3000 test/test_coverage.dart &
```
[the remark this answers is not in the document]
Comment (mannprerak2) [reply tool/coverage.sh:14]: Nope

File: bin/ffigen.dart [hunk 4]
```diff
@@ -1,211 +1,7 @@
 // Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
+//
+// @dart=2.7
```
[the remark this answers is not in the document]
Comment (mannprerak2) [reply bin/ffigen.dart:6]: Done

File: lib/src/executables/ffigen.dart (hunk 2 above)
Comment (mannprerak2) [reply lib/src/executables/ffigen.dart]: Yeah, I actually forgot to update the template.

**Comment by mannprerak2** [comment]
@dcharkes perhaps you should merge #125 first.

**Pull request merged**